src/pages/Books.js

```diff
@@ -36,48 +38,75 @@ export default function Books() {
       <div>
         {myBooks}
       </div>
+      <div className="Line" />
+      <form className="form-panel">
+        <div className="add-new">
+          ADD NEW BOOK
+        </div>
+        <div className="container-5">
+          <label htmlFor="author">
+            <input
+              id="author"
+              name="author"
+              type="text"
+              placeholder="Book author"
+              className="entry"
+              size="150"
+              onChange={(e) => {
+                setAuthor(e.target.value);
+              }}
+            />
+          </label>
 
-      <form>
-        <label htmlFor="author">
-          Author:
-          <input
-            id="author"
-            name="author"
-            type="text"
-            onChange={(e) => {
-              setAuthor(e.target.value);
-            }}
-          />
-        </label>
+          <label htmlFor="title">
+            <input
+              id="title"
+              name="title"
+              type="text"
+              placeholder="Book title"
+              className="entry"
+              size="100"
+              onChange={(e) => {
+                setTitle(e.target.value);
+              }}
+            />
+          </label>
+
+          <label htmlFor="category">
+            <input
+              id="category"
+              name="category"
+              type="text"
+              placeholder="Book category"
+              className="entry"
+              size="100"
+              onChange={(e) => {
+                setCategory(e.target.value);
+              }}
+            />
+          </label>
 
-        <label htmlFor="title">
-          Title:
-          <input
-            id="title"
-            name="title"
-            type="text"
-            onChange={(e) => {
-              setTitle(e.target.value);
+          <button
+            type="submit"
+            className="Rectangle-4"
+            onClick={(e) => {
+              e.preventDefault();
+              const id = uuid();
+              dispatch(addBook({
+                id, author, title, category,
+              }));
+              dispatch(addItemAxios({
+                id,
+                author,
+                title,
+                category,
+              }));
             }}
-          />
-        </label>
+          >
+            ADD BOOK
+          </button>
+        </div>
 
```

Comment:
Kindly fix the styling for the `Add Book` button. 
Your design ⬇️
![image](https://user-images.githubusercontent.com/33268927/224405574-bc1d28dc-40a2-4ffb-acc5-26823f2b9745.png)

Provided design ⬇️
![image](https://user-images.githubusercontent.com/33268927/224405828-8c730756-d5bf-4a63-9f09-c08d0f4eeb6b.png)